### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by MTecknology** [hlandau/acmetool#329]
@hlandau Per #195, it seems the appropriate license file was removed in favor of some non-standard (rilts) solution. Unfortunately, that rilts document makes some assumptions that are not correct, the most notable being that the file name is ambiguous. This is not true; that location is an accepted standard, including jurisdictions where English is not the native tongue.

This may lead to a tiny bit of convenience to you, but it causes headaches for everyone else that needs to audit the source.

For instance: https://salsa.debian.org/go-team/packages/golang-gopkg-hlandau-acmeapi.v2/blob/debian/sid/debian/patches/license.patch

Please consider adopting a well-known/standardized solution for keeping track of license information.

**Comment by rspier** [comment]
+1

LICENSE (or COPYING) files are the standard used in almost every piece of open source software.  

rilts has very little uptake or mindshare.

The lack of an obvious LICENSE (or License) can prevent some people from contributing because the legal terms aren't clear.

**Comment by Mrten** [comment]
Just FYI: You are asking the author of rilts not to use his own licensing ways.
https://github.com/hlandau/rilts/blob/master/RILTS.md

**Comment by MTecknology** [comment]
> Just FYI: You are asking the author of rilts not to use his own licensing ways.
> https://github.com/hlandau/rilts/blob/master/RILTS.md

Yes, it would be wise of them to adopt a standard licensing strategy.
